ogcapi-features/extensions/transaction/openapi.yaml

```diff
@@ -219,11 +278,6 @@ components:
             $ref: '../../../core/commons.yaml#/components/schemas/link'
         assets:
           $ref: '../../../core/commons.yaml#/components/schemas/assets'
-      example:
```

Comment:
I think this missed being deleted when this block was copied from the Item schema